forge/routes/api/team.js

```diff
@@ -484,12 +484,54 @@ module.exports = async function (app) {
         // That is handled by the beforeDestroy hook on the Team model and the
         // call to destroy the team will throw an error
         try {
+            const instanceCount = await request.team.instanceCount()
+            if (instanceCount > 0) {
+                // need to delete Instances
+                const instances = await app.db.models.Project.byTeam(request.team.hashid)
+                for (const instance of instances) {
+                    try {
+                        await app.containers.remove(instance)
+                    } catch (err) {
+                        if (err?.statusCode !== 404) {
+                            throw err
+                        }
+                    }
+
+                    // disconnect any devices from the Instance
```

Comment:
Given we're about to delete the devices, I don't think we should send them all a command that will cause them to call home and change their state.